mitosheet/src/components/taskpanes/ExcelRangeImport/ExcelRangeImportTaskpane.tsx

```diff
@@ -259,7 +284,94 @@ const ExcelRangeImportTaskpane = (props: ExcelRangeImportTaskpaneProps): JSX.Ele
                                     />
                                 </Col>
                             </Row>
+                            {range_import.type === 'upper left corner value' && 
+                                <>
+                                    <Row justify="space-between">
+                                        <Col>
+                                            <p>
+                                                Table End Condition
+                                            </p>
+                                        </Col>
+                                        <Col>
+                                            <Select
+                                                width="medium"
+                                                value={range_import.end_condition.type}
+                                                onChange={(newType) => {
+                                                    setParams((prevParams) => {
+                                                        const newRangeImports = window.structuredClone(prevParams.range_imports);
+                                                        const newRangeImport: ExcelRangeUpperLeftCornerValueImport = window.structuredClone(range_import);
+
+                                                        if (newType === 'first empty cell') {
```

Comment:
Maybe we can first cast `newType` to ExcelRangeImportType so we make sure these string checks remain consistent if we change the ExcelRangeImportType options. 


Can we also create a test to make sure that its kept in sync with the backend